addons/html_editor/static/src/main/feff_plugin.js

```diff
@@ -32,6 +32,8 @@ export class FeffPlugin extends Plugin {
         intangible_char_for_keyboard_navigation_predicates: (ev, char, lastSkipped) =>
             // Skip first FEFF, but not the second one (unless shift is pressed).
             char === "\uFEFF" && (ev.shiftKey || lastSkipped !== "\uFEFF"),
+        content_for_clipboard_processors: this.processContentForClipboard.bind(this),
+        clipboard_text_processors: (text) => text.replace(/\ufeff/g, ""),
```

Comment:
Just did an ulta quick pass since I already knew what was happening in this PR.

Those two resources should have similar names. I suggest `clipboard_content_processors` and `clipboard_text_processors`.

Reply:
(change pending, will do it after Testing BE)